### PR TITLE
Added new validators for GST& VoterID

### PR DESCRIPTION
**Pull request by ayushj98**
Implemented validators for GST and Voter ID based in India.

### CODE REVIEW

File: validators/GST_IN.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const dry = require('drytypes')
+
+module.exports = dry.makeDryType((x) => {
+    if (typeof x !== 'string') {
+        return { success: false, message: 'GST number must be a Alphanumeric String!' }
```
Comment (SwapnilSoni1999) [inline validators/GST_IN.js:5]: Alphanum check is also required along with `typeof` check

**Review by SwapnilSoni1999 — CHANGES_REQUESTED** [review]
please make required changes

File: validators/GST_IN.js (hunk 1 above)
Comment (ayushj98) [reply validators/GST_IN.js:5]: the regex below will check that ,so its not necessary over here imo

**Review by SwapnilSoni1999 — CHANGES_REQUESTED** [review]
Only test cases are needed can you please add them

**Comment by ayushj98** [comment]
> Only test cases are needed can you please add them

@SwapnilSoni1999  Added unit test cases , please have a look

**Review by SwapnilSoni1999 — APPROVED** [review]

**Comment by ayushj98** [comment]
@SwapnilSoni1999 can you add the tag for hacktoberfest ?

**Pull request merged**